### PR TITLE
Add animation Hacktoberfest

### PR DESCRIPTION
**Pull request by casiimir**
Wonderful!
Hacktoberfest 2020!

### CODE REVIEW

File: animations/orangy.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    font-size: 30px;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Staatliches', cursive;
+    letter-spacing: 10px;
+    word-spacing: 20px;  
+    opacity: 0.6;
+  }
```
Comment (NiallEccles) [inline animations/orangy.css:17]: Need to update this class to be your animation name.

File: animations/orangy.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+@keyframes example {
+    from {color: bown;}
```
Comment (NiallEccles) [inline animations/orangy.css:2]: Brown is misspelt.

File: animations/orangy.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    letter-spacing: 10px;
+    word-spacing: 20px;  
+    opacity: 0.6;
+  }
+  
+  .text {
+      animation-name: example;
+    animation-duration: 0.5s;
+    animation-iteration-count: infinite;  animation-direction: alternate;
+  }
```
Comment (NiallEccles) [inline animations/orangy.css:23]: These need to go in your class above.

File: animations/orangy.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  
+  
+  .header:hover {
+    opacity: 1;
+    background-color: black;
+   }
```
Comment (NiallEccles) [inline animations/orangy.css:29]: Need to update header to be your class name.

**Review by NiallEccles — CHANGES_REQUESTED** [review]
I have a few changes for you to make.

**Review by casiimir — COMMENTED** [review]
Thank you

**Review by casiimir — COMMENTED** [review]
It's all right?

File: animations/orangy.css [hunk 5]
```diff
@@ -0,0 +1,24 @@
+@keyframes animate {
+    from {color: brown;}
+    to {color: orange;}
+  }
+  
+  .orangy {
+    color: brown;
+    font-size: 30px;
+    height: 100vh;
```
Comment (NiallEccles) [inline animations/orangy.css:9]: Maybe remove the height?

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Yes! Looks good. I would just maybe say if you could remove the height from your class? Just because if someone visits the page and your animation is selected then it pushes the select field down and then the user has to scroll to select a different animation.

File: animations/orangy.css (hunk 5 above)
Comment (casiimir) [reply animations/orangy.css:9]: Oh sure!

**Review by casiimir — COMMENTED** [review]
Remove height 👍

**Comment by casiimir** [comment]
👍

**Pull request merged**